### DISCUSSION TITLE
请教一下composition中播放多个pagFile渲染问题

### DISCUSSION THREAD

**Discussion by newbeex** [Tencent/libpag#2079] [Q&A]
你好，我在web端使用pagComposition，动态添加了一些pagFile，pagComposition.addLayer(pagFile);
这个pagFile中存在bmp预合成，我的问题是：当浏览器卡顿或动画卡顿的时候，composition中的layer，会显示出一个灰色半透明的背景，正常显示时没有问题的，背景是透明色。研究了几天也没搞清楚如何解决，烦请指教。
![27404ff1fcf1f30a5fe8210874495baf](https://github.com/Tencent/libpag/assets/3202913/959a94de-29b3-4b1c-a735-7560b991c97d)

**Comment by kevingpqi123** [comment]
这类问题建议构建一个复现的 demo， 仅仅靠描述难以定位问题

**Comment by CodeJhF** [comment]
同类问题已由 #3101 继续跟踪，为避免重复排查，先关闭本条。
如果有新的复现信息，欢迎到该帖补充，或在本帖回复后重开。（stale-close）
